torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -634,11 +726,36 @@ struct CodeImpl {
     }
   }
 
+  void emitLoadInputs(at::ArrayRef<Value*> inputs, int num_include) {
+    int count = 0;
+    for (Value* input : inputs) {
+      if (count < num_include) {
+        emitUse(input, false);
+        count++;
+      } else {
+        emitUse(input, true);
+      }
+    }
+  }
+
   void emitOperator(Node* node) {
-    emitLoadInputs(node->inputs());
     const Operator& op = node->getOperator();
+    auto unique_op_name = op.schema().overload_name() != ""
```

Comment:
Though the overhead added here is low, ideally we should still avoid it for normal TorchScript execution. One way to do so is to subclass interpreter for mobile export. The subclass should override emitOperator with an implementation with additional logic about ignoring arguments. This way, normal TorchScript execution would not be slowed down at all. What do you think?